homeassistant/components/goalzero/const.py

```diff
@@ -25,6 +56,92 @@
     "inputDetected": ["Input Detected", DEVICE_CLASS_POWER, None],
 }
 
+SENSOR_DICT = {
+    "wattsIn": {
+        ATTR_NAME: "Watts In",
+        ATTR_DEVICE_CLASS: DEVICE_CLASS_POWER,
+        ATTR_UNIT_OF_MEASUREMENT: POWER_WATT,
+        ATTR_LAST_RESET: datetime.fromtimestamp(0),
```

Comment:
This is not a sum? Why does it have a last_reset?

Reply:
Prehaps I don't understand the state class. This is the realtime watts that are going into the device.

Reply:
So there are 2 types of measurement we support right now. The current value (readtime Watts)
and counter values (e.g., Total amount of energy in kWh produced since X).

Both those cases can get a `STATE_CLASS_MEASUREMENT`.

For counter/sum/total values HA need to know how much energy is used in X period, it needs to know when the counter was last reset (e.g., Daily stats reset every day, so, provide the timestamp of when it was reset).

As this is not a sum, but just a current value, is should not have a `last_reset`.

Reply:
Turns out these are all realtime counters.